dp-identity-data-migration/userdataextraction/user_extraction.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+
+	"os"
+
+	"time"
+
+	"github.com/ONSdigital/dp-zebedee-sdk-go/zebedee"
+	"github.com/google/uuid"
+)
+
+type config struct {
+	filename,
+	host,
+	pword,
+	user string
+}
+
+func readConfig() (conf config) {
+	for _, e := range os.Environ() {
+		pair := strings.SplitN(e, "=", 2)
+		if pair[0] == "filename" {
+			conf.filename = pair[1]
+		}
+		if pair[0] == "zebedee_user" {
+			conf.user = pair[1]
+		}
+		if pair[0] == "zebedee_pword" {
+			conf.pword = pair[1]
+		}
+		if pair[0] == "zebedee_host" {
+			conf.host = pair[1]
+		}
+	}
+	if conf.host == "" || conf.pword == "" || conf.user == "" || conf.filename == "" {
+		fmt.Println("Please set Environment Variables ")
+		os.Exit(1)
+	}
+
+	return conf
```

Comment:
Since you're declaring the return variable as part of the function definition, you can simply do a `return` here instead of `return conf`